mod-orders/mod-orders.postman_collection.json

```diff
@@ -0,0 +1,901 @@
+						"header": [
+							{
+								"key": "X-Okapi-Tenant",
+								"value": "{{xokapitenant}}"
+							},
+							{
+								"key": "Content-Type",
+								"value": "application/json"
+							},
+							{
+								"key": "x-okapi-token",
+								"value": "{{xokapitoken}}"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n\t\n}"
+						},
+						"url": {
+							"raw": "{{protocol}}://{{url}}:{{okapiport}}/orders",
+							"protocol": "{{protocol}}",
+							"host": [
+								"{{url}}"
+							],
+							"port": "{{okapiport}}",
+							"path": [
+								"orders"
+							]
+						},
+						"description": "GET /orders/ requests that return 200"
+					},
+					"response": []
+				},
+				{
+					"name": "POST /orders - create complete order",
+					"event": [
+						{
+							"listen": "prerequest",
+							"script": {
+								"id": "eb2052a4-e388-4860-80be-ea96976bcf21",
+								"exec": [
+									"pm.sendRequest({",
+									"        url: pm.variables.get(\"mod-ordersResourcesURL\") + \"/po_listed_print_monograph.json\",",
+									"        method: \"GET\"",
+									"    },",
+									"    function (err, res) {",
+									"        pm.globals.set(\"po_listed_print_monograph.json\", res.text());     ",
+									"    }",
+									");",
+									"",
+									"",
+									"",
+									"",
+									""
+								],
+								"type": "text/javascript"
+							}
+						},
+						{
+							"listen": "test",
+							"script": {
+								"id": "3bf00f6d-8798-4764-a256-2d360ea02876",
+								"exec": [
+									"var jsonData = pm.response.json();",
+									" ",
+									"pm.environment.set(\"order_id\", jsonData.id);",
+									"",
+									"pm.test(\"Status code is 201\", function () {",
+									"    pm.response.to.have.status(201);",
+									"});",
+									"",
+									"pm.test(\"po_lines exist\", function () {",
+									"    pm.expect(jsonData.po_lines).to.have.lengthOf(1);",
+									"});",
+									"",
+									"pm.test(\"Each order has these optional fields\", function(){",
+									"    pm.expect(jsonData.id).to.exist;",
+									"    pm.expect(jsonData.approved).to.exist;",
+									"    pm.expect(jsonData.po_number).to.exist;",
+									"    pm.expect(jsonData.notes).to.exist;",
+									"    pm.expect(jsonData.total_items).to.exist;",
+									"    pm.expect(jsonData.vendor).to.exist;",
+									"});",
+									""
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "X-Okapi-Tenant",
+								"value": "{{xokapitenant}}"
+							},
+							{
+								"key": "Content-Type",
+								"name": "Content-Type",
+								"value": "application/json",
+								"type": "text"
+							},
+							{
+								"key": "x-okapi-token",
+								"value": "{{xokapitoken}}"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{{po_listed_electronic_monograph}}"
```

Comment:
Either this or pm.globals.set(\"po_listed_print_monograph.json\") need to be changed to set and fetch from the same variable

Reply:
Corrected to `po_listed_electronic_monograph`